src/mem/external_slave.hh

```diff
@@ -123,17 +132,31 @@ class ExternalSlave : public SimObject
     static std::map<std::string, Handler *> portHandlers;
 
   public:
+    /**
+     * @ingroup api_port
+     */
     ExternalSlave(ExternalSlaveParams *params);
 
-    /** Port interface.  Responds only to port "port" */
+    /**
+     * Port interface. Responds only to port "port".
+     *
+     * @ingroup api_port
+     */
     Port &getPort(const std::string &if_name,
```

Comment:
This should be defined in the sublcass, not here (I think). @BobbyRBruce, do you remember how APIs worked with sublcasses last time?

Reply:
done